proparse/src/main/java/org/prorefactor/proparse/antlr4/PreproEval.java

```diff
@@ -791,7 +794,7 @@ static String string(Object a) {
     return a.toString();
   }
 
-  static String substring(Object a, Object b, Object c) {
+  static String substring(Object a, Object b, Object c, SubstringType type) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "type". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1172)